types/project.go

```diff
@@ -119,6 +119,36 @@ func (p *Project) ConfigNames() []string {
 	return names
 }
 
+func (p *Project) ServicesBuild() []string {
```

Comment:
For even more flexibility we could introduce `ServicesWith(predicate func(ServiceConfig) bool) []string`

By the way, makes me wonder we only want service names here, or actual services. Could have both by introducing 
`ServicesWith(predicate ...)` and `ServiceNamesWith(predicate ...)`, wdyt ?

Reply:
fixed, could you please review?